portfolio/src/main/webapp/style.css

```diff
@@ -2,8 +2,18 @@
   margin-left: auto;
   margin-right: auto;
   width: 650px;
+  
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+img{
```

Comment:
Nice! This will apply to all images though, do you want that? In css, you can specify that you want a styling to apply to only select classes or ids.